packages/subgraph/.gitignore

```diff
@@ -0,0 +1,19 @@
+# Geenrated types
+generated/
```

Comment:
it seems you changed the `codegen` to output the generated files to `src/types` so this isn't needed anumore.